llvm/test/Instrumentation/MemorySanitizer/AArch64/or-shadow-for-strict-instructions.ll

```diff
@@ -0,0 +1,54 @@
+; NOTE: Assertions have been autogenerated by utils/update_test_checks.py UTC_ARGS: --version 4
+; RUN: opt < %s -passes=msan -S                                         | FileCheck %s --check-prefixes=CLEAN
+; RUN: opt < %s -passes=msan -S -msan-or-shadow-for-strict-instructions | FileCheck %s --check-prefixes=OR-SHADOW
```

Comment:
this tests only non-recover mode where the patch is no-op?

Reply:
I've added recover mode test output.

(Though the conceptual no-op mode is actually the more useful mode so far.)